pyxdsm/XDSM.py

```diff
@@ -409,6 +415,8 @@ def _build_node_grid(self):
             style = out.style
             if out.stack:
                 style += ",stack"
+            if out.faded is True:  # fading
```

Comment:
You don't need to check `is True` with boolean comparison.

Reply:
This is true